### DISCUSSION TITLE
How to Return cookie value from cookie_jar file as string

### DISCUSSION THREAD

**Discussion by MohdRashid01** [kevinzg/facebook-scraper#520] [Q&A] [answered]
Hi All

I have one file that name is cookie_jar file in that file there is a cookie value stored in it that i need return in it as string way i have written one command but dont know how to put in as a bash script or shell script. So that my script should return as a token values. Here is the code

cookie_value = "'cookie: PF='$PF'; p_lplcwmsk='$p_lplcwmsk'; BIGipServerNP_QA_QFLOGINQAS0_7011='$BIGipServerNP_QA_QFLOGINQAS0_7011'; aatoken='$aatoken'; UserName='$UserName'; Auth='$Auth'; XSRF-TOKEN='$XSRF-TOKEN'; z='$z'; BN20='$BN20'; UserFirmId='$UserFirmId'; TS01a1bccb='$TS01a1bccb'; TS01e14722='$TS01e14722';"

echo cookie_value

please tell me how to write in script so that it can return string from cookie jar file as cookie value i want it that only from script

**Comment by neon-ninja** [comment, marked as answer]
Duplicate of https://github.com/kevinzg/facebook-scraper/issues/521

**Reply by MohdRashid01** [reply]
how to delete my post can u tell me
